package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
should this be modified? the changes here seem unrelated (and quite a lot).

Reply:
Reverted.